CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Changed
+- Use logger from k8s authenticator client; its timestamp format contains milliseconds precision.
+[cyberark/conjur-authn-k8s-client#221](https://github.com/cyberark/conjur-authn-k8s-client/issues/221)
```

Comment:
please indent this. also the link is broken.

are we changing the logger so that its timestamp format contains milliseconds precision or is this just a side-effect of the change?

Reply:
is this the only customer facing change between the loggers?

Reply:
The milliseconds is the customer facing benefit of this change.